country/israel.md

```diff
@@ -1,21 +1,16 @@
 # Israel
-
-Israel’s government uses Section 30A of the Communications Broadcasting Law to regulate the sending of unsolicited commercial emails. The amendment was introduced in December 2008 and dictates that an advertiser may not send an advertisement by fax, automatic dialing system, electronic message (email) or SMS, without obtaining the recipient’s explicit consent in advance and in writing.
+Israel's government uses Section 30A of the Communications Broadcasting Law to regulate the sending of unsolicited commercial emails. The amendment was introduced in December 2008 and dictates that an advertiser may not send an advertisement by fax, automatic dialing system, electronic message (email) or SMS, without obtaining the recipient's explicit consent in advance and in writing.
 
 ## Content required
-
 The law sets out that those who want to send an advertising email must clearly highlight it is an advert and include your name, address and contact details. The recipient must also be able to unsubscribe from those emails.
 
 ## Consent
-
 You can only send marketing emails to those who have provided explicit consent to receive marketing emails from you.
 
 The recipient can also be sent marketing emails if they shared their contact info, when purchasing a service or product, or when negotiating such purchase (specified for general advertising, which includes marketing emails)
 
 ## Penalties
-
-The fine that would be imposed would depend on the severity of breach of the law and could go up to ILS 202,000 in a criminal court. Recipients of spam email may also seek damages which in a civil court could be as much as ILS 1,000 for each message sent by the advertiser to the recipient.
+The fine that would be imposed would depend on the severity of breach of the law and could go up to 202,000 NIS in a criminal court. Recipients of spam email may also seek damages which in a civil court could be as much as ILS 1,000 for each message sent by the advertiser to the recipient.
```

Comment:
> could go up to 202,000 NIS in a 

It's new Israeli shekel (NIS) but the ISO is ILS. Maybe worth checking we're consistent in use as they are interchangeable.

Reply:
Good spot. [gov.il](https://www.gov.il/en/pages/17052018_7) references ILS so I've gone with that. 

https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/2d3e1198005a5df49c80725c34536ea9fa70b684